test/utils/verifyExpirationTime.test.ts

```diff
@@ -41,7 +41,7 @@ describe("Test verifyExpirationTime",  function() {
             "0",
             "0"];
 
-        testData.in = {enabled: "1", claim: claim, timestamp: 1669884009};
+        testData.in = {enabled: "0", claim, timestamp: 1669884009};
```

Comment:
why have you changed this flag?

Reply:
for this unit test we have `expiration flag 0`, in prev implementation of `verifyExpirationTime` we parced this flag from `header.claimFlags` and now we pass it as separate input `  signal input enabled; // claimFlags[3] (expiration flag) is set`